### ISSUE TITLE
Is the RP-Initiated Logout scenario already implemented?

### ISSUE THREAD

**Issue by ngriesser** [mitreid-connect/OpenID-Connect-Java-Spring-Server#856]
Setting the post_logout_redirect_uri in the database does not have any effect, calling the logout endpoint with a post_logout_redirect_uri parameter redirects to the sso server itself.

**Comment by jricher** [comment]
The logout endpoint is not meant to be called externally. RP-initiated logout is not implemented yet.

**Comment by ngriesser** [comment]
So to implement it the right way, we should add another logout endpoint that asks the user for consent and eventually calls the internal logout. Am I getting it right?

**Comment by jricher** [comment]
Correct, and it should offer the users the option of logging out of the IdP and potentially sending logout messages to all other RPs. The internal logout URL really should have some CSRF protection on it anyway to prevent external callers from doing a drive-by logout.
